tripy/CONTRIBUTING.md

```diff
@@ -105,12 +105,43 @@ Please make sure any contributions you make satisfy the developer certificate of
 >		this project or the open source license(s) involved.
 
 
+### Use custom MLIR-TensorRT with Tripy
+Tripy depends on [MLIR-TensorRT](../mlir-tensorrt/README.md) for compilation and execution.
+Tripy container currently builds with [mlir-tensorrt-v0.1.29](https://github.com/NVIDIA/TensorRT-Incubator/releases/tag/mlir-tensorrt-v0.1.29), but you may
+choose to test Tripy with custom version of MLIR-TensorRT.
+
+1. [Build custom MLIR-TensorRT](#contributing-to-mlir-tensorrt)
+
+2. Launch the container; from the [`tripy` root directory](.) with mlir-tensorrt repository mapped for accessing wheels files, run:
+    ```bash
+    docker run --gpus all -it -v $(pwd):/tripy/ -v $(pwd)/../mlir-tensorrt:/mlir-tensorrt  --rm tripy:latest
+    ```
+
+3. Install MLIR-TensorRT wheels
+    MLIR-TensorRT builds with a specific version of TensorRT, ensure it matches the version required by Tripy.
+    This requirement can be relaxed in future.
```

Comment:
@parthchadha Looks like we build MLIR-TensorRT container using [mlir-tensorrt-compiler==0.1.29+cuda12.trt102](https://github.com/NVIDIA/TensorRT-Incubator/blob/main/tripy/pyproject.toml#L10) and install `TensorRT-10.1.0.27` in the [Dockerfile](https://github.com/NVIDIA/TensorRT-Incubator/blob/main/tripy/Dockerfile#L28).

Reply:
That's intentional. We build `mlir-tensorrt` with 10.2 and can link with 10.0+ (linking with 10.1 to keep it compatible with TRT-LLM).